### PR TITLE
K5.0 bootstrap datepicker

### PR DESCRIPTION
**Pull request by xillibit**

### CODE REVIEW

File: components/com_kunena/site/template/crypsisb3/layouts/topic/edit/editor/default.php [hunk 1]
```diff
@@ -178,7 +177,9 @@
 						alt="<?php echo JText::_('COM_KUNENA_POLL_REMOVE_POLL_OPTION'); ?>"> </i>
 					<br>
 					<label class="kpoll-term-lbl" for="kpoll-time-to-live"><?php echo JText::_('COM_KUNENA_POLL_TIME_TO_LIVE'); ?></label>
-					<?php echo JHtml::_('calendar', isset($this->poll->polltimetolive) ? $this->escape($this->poll->polltimetolive) : '0000-00-00', 'poll_time_to_live', 'kpoll-time-to-live', '%Y-%m-%d', 'class="form-control"'); ?>
+					<div class="input-append date">
+						<input type="text" class="span12" name="poll_time_to_live" data-date-format="mm/dd/yyyy"><span class="add-on"><i class="icon-grid-view-2 "></i></span>
```
Comment (810) [inline components/com_kunena/site/template/crypsisb3/layouts/topic/edit/editor/default.php]: should be col-md-12 instead of span12

Comment (xillibit) [inline components/com_kunena/site/template/crypsisb3/layouts/topic/edit/editor/default.php]: only on crypsis b3 ?

Comment (810) [inline components/com_kunena/site/template/crypsisb3/layouts/topic/edit/editor/default.php]: yes.

b3 = col-md-*
b2 = span*

Comment (xillibit) [inline components/com_kunena/site/template/crypsisb3/layouts/topic/edit/editor/default.php]: Updated

**Comment by 810** [comment]
Could you also add the datepicker on edit announcements

**Pull request merged**